site/app/controllers/course/CourseMaterialsController.php

```diff
@@ -461,6 +461,9 @@ public function ajaxUploadCourseMaterialsFiles(): JsonResponse {
         if (isset($_POST['sections']) && $sections_lock) {
             $sections = $_POST['sections'];
             $sections_exploded = @explode(",", $sections);
+            if ($sections_exploded[0] == "") {
```

Comment:
It would be better to use `===` here as well

Reply:
fixed